src/main/java/graphql/introspection/IntrospectionResultToSchema.java

```diff
@@ -264,7 +264,7 @@ private void createDeprecatedDirective(Map<String, Object> field, NodeDirectives
         for (Map<String, Object> arg : args) {
             Type argType = createTypeIndirection((Map<String, Object>) arg.get("type"));
             InputValueDefinition.Builder inputValueDefinition = InputValueDefinition.newInputValueDefinition().name((String) arg.get("name")).type(argType);
-            inputValueDefinition.comments(toComment((String) arg.get("description")));
+            inputValueDefinition.description(toDescription((String) arg.get("description")));
```

Comment:
Since these are all

`toDescription((String) input.get("description")`

you could have `String toDescription(Map<String, Object> input)` say - much less code

Reply:
Yep, that makes a lot of sense